pybamm/models/submodels/electrolyte_diffusion/base_electrolyte_diffusion.py

```diff
@@ -103,6 +106,17 @@ def _get_standard_flux_variables(self, N_e):
 
         return variables
 
+    def set_boundary_conditions(self, variables):
```

Comment:
I think we should put the boundary conditions function in each of the individual submodels. I know it repeats more code, but I think it is much easier to figure out what is happening in a submodel.